apps/yggdrasil/test/test_helper.exs

```diff
@@ -1,4 +1,8 @@
 ExUnit.start()
 
-Mix.Task.run "ecto.create", ~w(-r Yggdrasil.Repo --quiet)
-Mix.Task.run "ecto.migrate", ~w(-r Yggdrasil.Repo --quiet)
+repo = ~w(-r Yggdrasil.Repo --quiet)
+
+Mix.Task.run "ecto.drop", repo
+Mix.Task.run "ecto.create", repo
+Mix.Task.run "ecto.migrate", repo
+Mix.Task.run "run", ~w(apps/yggdrasil/priv/repo/seeds.exs)
```

Comment:
Actually, the seed file could check to that point so the drop should not be needed actually. You have full access to the repo.

To answer your question migrations can seed data as well, but it's via sql not through the repo if I'm not mistaken.

So the only issue here is that the `run` path is only valid at the root of the project, and if you are in the sub app it breaks. So either the run needs to be moved or put in the travis file or a better way to get the sub app path is needed which there may be. Didn't have a lot of time to look at it.

The reason that I think it's important to be bale to cd down into the sub app and run mix test, is because sometimes I find it useful to run one test like

```
mix test test/yggdrasil/user_test.exs
```

with the path like it is it breaks.